invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/uploads_items/ComputerTabletUploadsItem.js

```diff
@@ -0,0 +1,143 @@
+  const subjects = _get(result, "ui.subjects", []);
+  const createdDate = _get(
+    result,
+    "ui.created_date_l10n_long",
+    i18next.t("No creation date found.")
+  );
+  const publicationDate = _get(
+    result,
+    "ui.publication_date_l10n_long",
+    i18next.t("No publication date found.")
+  );
+  const resource_type = _get(
+    result,
+    "ui.resource_type.title_l10n",
+    i18next.t("No resource type")
+  );
+  const version = result.ui?.version;
+  const is_published = result.is_published;
+
+  const viewLink = is_published
+    ? `/records/${result.id}`
+    : `/uploads/${result.id}`;
+  const icon = is_published ? (
+    <Icon name="check" color="green" />
+  ) : (
+    <Icon name="upload" color="red" />
+  );
+
+  return (
+    <Item
+      key={index}
+      className="deposits-list-item mb-20 computer tablet only flex"
+    >
+      <div className="status-icon mr-10">
+        <Item.Content verticalAlign="top">
+          <Item.Extra>{icon}</Item.Extra>
+        </Item.Content>
+      </div>
+      <Item.Content style={{ cursor: "default" }}>
+        <Item.Extra className="labels-actions">
+          {result.status in statuses && result.status !== "published" && (
+            <Label size="tiny" color={statuses[result.status].color}>
+              {statuses[result.status].title}
+            </Label>
+          )}
+          <Label size="tiny" color="blue">
+            {publicationDate} ({version})
+          </Label>
+          <Label size="tiny" color="grey">
+            {resource_type}
+          </Label>
+          <Label size="tiny" className={`access-status ${access_status_id}`}>
+            <i className={`icon ${access_status_icon}`}></i>
+            {access_status}
+          </Label>
+          <Button
+            compact
+            size="small"
+            floated="right"
+            onClick={() => editRecord()}
+          >
+            <Icon name="edit" />
+            {i18next.t("Edit")}
+          </Button>
+          {is_published && (
+            <Button compact size="small" floated="right" href={viewLink}>
+              <Icon name="eye" />
+              {i18next.t("View")}
+            </Button>
+          )}
+        </Item.Extra>
+        <Item.Header as="h2">
+          <a href={viewLink} className="truncate-lines-2">
+            {title}
+          </a>
+        </Item.Header>
+        <Item.Meta>
+          <div className="creatibutors">
+            <SearchItemCreators creators={creators} />
+          </div>
+        </Item.Meta>
+        <Item.Description>
+          {_truncate(description_stripped, {
+            length: 350,
+          })}
+        </Item.Description>
+        <Item.Extra>
+          {subjects.map((subject, index) => (
+            <Label key={index} size="tiny">
+              {subject.title_l10n}
+            </Label>
+          ))}
+          <div>
+            <small>
+              {i18next.t("Uploaded on")} <span>{createdDate}</span>
```

Comment:
In case `createdDate` is not defined, you might  end up with a label like:

`Uploaded on No creation date found.`

Is it intended?